Servlet.txt

```diff
@@ -0,0 +1,35 @@
+package com.servlet;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+
+@WebServlet("/myAge")
+public class AgeServlet extends HttpServlet {
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Lipseste implementarea pentru cazul in care data nasterii nu este data ca parametru de query in request.